Quantum-Control-Applications-QuAM/Superconducting/configuration/make_quam.py

```diff
@@ -133,21 +140,27 @@ def create_quam_superconducting(
                 depletion_time=1 * u.us,
             ),
         )
-
+        # Add the transmon to the machine
         machine.qubits[transmon.name] = transmon
+        # Add the transmon to the active qubits
         machine.active_qubit_names.append(transmon.name)
-
+        # Add the set of default pulses
+        # TODO: improve this function
         add_default_transmon_pulses(transmon)
-
+        # Set the Octave
+        # TODO: why not doing it directly in octave.initialize_frequency_converters(output_mode="always_on", LO_frequency=4.7e9, gain=0)
         RF_output = transmon.xy.frequency_converter_up
         RF_output.channel = transmon.xy.get_reference()
         RF_output.output_mode = "always_on"
         # RF_output.output_mode = "triggered"
         RF_output.LO_frequency = 4.7 * u.GHz
+        # TODO: why? the LO should be in the config and the Octave will automatically be set to it when opening a QM
```

Comment:
I thought I'd add this because the LO frequency is almost certainly not the one we want to use, so it would need to be changed afterwards. But please change if you think it's unnecessary, I'm all for reducing the print statements